README.md

```diff
@@ -16,7 +16,14 @@ Lightly is a computer vision framework for self-supervised learning.
 - [Github](https://github.com/lightly-ai/lightly)
 - [Discord](https://discord.gg/xvNJW94)
 
-### Supported Models
+### Features
+
+- modular framework
```

Comment:
Either add a sentence above, i.e. "Lightly offers features like" and then keep the small letters or change the first letter of every line to capital